src/rebar_prv_dialyzer.erl

```diff
@@ -39,24 +40,33 @@ desc() ->
     "This command will build, and keep up-to-date, a suitable PLT and will use "
     "it to carry out success typing analysis on the current project.\n"
     "\n"
-    "The following (optional) configurations can be added to a rebar.config:\n"
-    "`dialyzer_warnings` - a list of dialyzer warnings\n"
-    "`dialyzer_plt` - the PLT file to use\n"
-    "`dialyzer_plt_apps` - a list of applications to include in the PLT file*\n"
-    "`dialyzer_plt_warnings` - display warnings when updating a PLT file "
-    "(boolean)\n"
-    "`dialyzer_base_plt` - the base PLT file to use**\n"
-    "`dialyzer_base_plt_dir` - the base PLT directory**\n"
-    "`dialyzer_base_plt_apps` - a list of applications to include in the base "
-    "PLT file**\n"
+    "The following (optional) configurations can be added to a `proplist` of "
+    "options `dialyzer` in rebar.config:\n"
+    "`warnings` - a list of dialyzer warnings\n"
+    "`get_warnings` - display warnings when altering a PLT file (boolean)\n"
+    "`plt_extra_apps` - a list of applications to include in the PLT file*\n"
+    "`plt_location` - the location of the PLT file, `local` to store in the "
+    "profile's base directory (default) or a custom directory.\n"
+    "`plt_prefix` - the prefix to the PLT file, defaults to \"rebar3\"**\n"
+    "`base_plt_apps` - a list of applications to include in the base "
+    "PLT file***\n"
+    "`base_plt` - the base PLT file to use***\n"
+    "`base_plt_location` - the location of base PLT file, `global` to store in "
+    "$HOME/.cache/rebar3 (default) or  a custom directory***\n"
     "\n"
+    "For example, to warn on unmatched returns: \n"
+    "{dialyzer, [{warnings, [unmatched_returns]}]}.\n"
+    "\n"
+    "`base_plt_prefix` - the prefix to the base PLT file, defaults to "
```

Comment:
Messed up the docs here I need to move this above the example and delete `base_plt` entry.